lib/holdings/spine.rb

```diff
@@ -21,12 +21,12 @@ def item
 
     # create sorting key for spine
     # shelfkey asc, then by sorting title asc, then by pub date desc
-    # note: pub_date must be inverted for descending sort
+    # note: pub_year_ss must be inverted for descending sort
```

Comment:
I thought `pub_year_ss` possibly contains non-numeric data, like date ranges and stuff. I don't think it's a drop-in replacement?

Reply:
`pub_date` contains non-numeric data.
<img width="290" height="98" alt="Screenshot 2025-10-03 at 3 46 21 PM" src="https://github.com/user-attachments/assets/0e1d8a04-3dde-49a1-a24f-98c57c84ea0c" />

https://sul-solr-prod-a.stanford.edu/solr/#/searchworks-prod/query?q=pub_year_tisim:*&q.op=OR&indent=true&fl=pub_date_search,%20pub_date,%20pub_year_isi,%20pub_year_tisim,%20pub_year_ss